### ISSUE TITLE
[DEL] - JupyterLab should be navigable by keyboard alone.

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#6]
The purpose of this WP is to move towards making JLab navigable through a keyboard.

This includes tasks such as #134 

```[tasklist]
### Related issues and tasks
- [ ] #168 
- [ ] #167 
- [ ] #169 
- [ ] #134 
- [ ] https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/182
```

**Comment by trallard** [comment]
@gabalafou  is working on fixing tab traps
